pkg/apiserver/handlers/webhook/mutation_labels.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhook
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	admv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+)
+
+type jsonPatchOperation string
+
+const (
+	jsonPatchReplaceOp jsonPatchOperation = "replace"
+	// LabelMetadataName is a well known reserved label key used by Antrea to store the resource's name
+	// as a label value.
+	LabelMetadataName  = "antrea.io/metadata.name"
+	antreaSvc          = "antrea"
+	antreaSvcNamespace = "kube-system"
```

Comment:
maybe this should not be hardcoded, but we should follow the same approach as in other places to determine the namespace

for example:
https://github.com/vmware-tanzu/antrea/blob/78b484c3fc12cb5c376c432c5a3e33c1ee248bcf/pkg/apiserver/certificate/cacert_controller.go#L77-L85

Reply:
done